sympy/ntheory/ecm.py

```diff
@@ -214,20 +214,21 @@ def _ecm_one_factor(n, B1=10000, B2=100000, max_curve=200):
         sigma = rgen.randint(6, n - 1)
         u = (sigma*sigma - 5) % n
         v = (4*sigma) % n
-        diff = v - u
         u_3 = pow(u, 3, n)
 
         try:
-            C = (pow(diff, 3, n)*(3*u + v)*mod_inverse(4*u_3*v, n) - 2) % n
+            # We use elliptic curve y**2 = x**3 + a*x**2 + x
+            # where a = pow(v - u, 3, n)*(3*u + v)*mod_inverse(4*u_3*v, n) - 2
+            # It is not necessary to retain a, only a24.
```

Comment:
Maybe write "We use the elliptic curve ..." 

You should also explain what a24 is.

Reply:
Comment has been modified.